### ISSUE TITLE
Support blacklisting specific vendor prefixes (-moz-available is bugged preventing fallback)

### ISSUE THREAD

**Issue by polarathene** [styled-components/styled-components#2512]
This perhaps falls more on stylis if styled-components continues to rely on that for handling this functionality, although development appears to have stalled there(nothing since Nov 2018), reaching parity for certain handling like CSS Grids like Autoprefixer does has been an issue since mid 2017, among other issues.

Presently, I want to implement a sticky footer, but `min-height: 100vh` doesn't quite work the same on mobile like it does with desktop, resulting in the footer being clipped a bit(requiring to scroll, even if there is plenty of empty space in the document above). 

I resolved this with `min-height: stretch` (also known as `fill-available`), this gets prefixed for Firefox as `-moz-available`, however there's been a bug for that support for 9 years where it only supports `width` props not `height`, and still doesn't work today in version 66 of Firefox(despite what caniuse states).

Unfortunately, I don't have control over prefixing when using styled-components. Otherwise I could prevent this prefix which is still recognized and applied by Firefox, overriding the fallback, breaking the layout.

---

**TL;DR:** `-moz-available` is a prefix for `stretch`/`fill-available` values, but is only supported by Firefox for width properties not height properties. The browser still recognizes the prefix and applies it preventing any fallback and breaking layout. Thus you cannot reliably use this support for Chrome and other supporting browsers.

[Stylis issue](https://github.com/thysultan/stylis.js/issues/94)

**Comment by probablyup** [comment]
Yeah it's more on the stylis side. There are [various ways](https://stackoverflow.com/questions/952861/targeting-only-firefox-with-css) to target only FF for css which you can use to override this particular unwanted style.
